data/reusables/dependabot/supported-package-managers.md

```diff
@@ -26,7 +26,7 @@ poetry         | `pip`            | v1               | | **✓** | |
 Terraform      | `terraform`      | >= 0.13, <= 1.0  | **✓** | **✓** | |
 yarn           | `npm`            | v1               | **✓** | **✓** | |
 
-[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle` and `build.gradle.kts` (for Kotlin projects).
+[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle`, `build.gradle.kts` (for Kotlin projects) and files included via the `apply` declaration that have `dependencies` in the filename.
```

Comment:
```suggestion
[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle`, `build.gradle.kts` (for Kotlin projects) and files included via the `apply` declaration that have `dependencies` in the filename. Note that `apply` does not support `apply to`, recursion, or advanced syntaxes (e.g. Kotlin's `apply` with `mapOf`, filenames defined by property).
```

Reply:
Adding caveats in https://github.com/dependabot/dependabot-core/pull/4155#issuecomment-918500360